dial_sync_test.go

```diff
@@ -8,6 +8,8 @@ import (
 	"time"
 
 	peer "github.com/libp2p/go-libp2p-peer"
+
```

Comment:
any reason for a new line here?

Reply:
import grouping (std, remote, local).

Reply:
Oh, wait, that's wrong. Should have been std, local, remote (fixed).